testhelpers/docker_registry.go

```diff
@@ -119,6 +126,11 @@ func Custom(handler http.Handler, permissions map[string]ImagePrivileges) http.H
 	})
 }
 
+// Start creates a docker registry following these rules:
+// - Shared handler will be use, otherwise a new one will be created
+// - by default the shared handler will be wrapped using a read only handler
+// - In case credentials are configured, the shared handler will be wrapped with a basic authentication handler and
+//   in any image privileges were set, then custom handler will be use to wrap the auth handler.
```

Comment:
```suggestion
//   if any image privileges were set, then the custom handler will be used to wrap the auth handler.
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1a4373ac85d225b6bd3771c3ccadae2a239cf6db)